pkgs/by-name/zo/zoom-us/package.nix

```diff
@@ -153,14 +156,14 @@ stdenv.mkDerivation rec {
     runHook postInstall
   '';
 
-  postFixup =  lib.optionalString stdenv.hostPlatform.isDarwin ''
+  postFixup = lib.optionalString stdenv.hostPlatform.isDarwin ''
     makeWrapper $out/Applications/zoom.us.app/Contents/MacOS/zoom.us $out/bin/zoom
   '' + lib.optionalString stdenv.hostPlatform.isLinux ''
     # Desktop File
     substituteInPlace $out/share/applications/Zoom.desktop \
         --replace-fail "Exec=/usr/bin/zoom" "Exec=$out/bin/zoom"
 
-    for i in aomhost zopen zoom ZoomLauncher ZoomWebviewHost; do
+    for i in aomhost zopen zoom ZoomLauncher ${webviewHostFixup "ZoomWebviewHost"}; do
```

Comment:
Does the build fail because `ZoomWebviewHost` doesn’t exist in older versions? If so, would it be simpler to add a `[ -f $out/opt/zoom/$i ]` condition?

Reply:
Yeah, that's actually a better idea 👍🏽